### PR TITLE
communication section in conduct 

### PR DESCRIPTION
**Pull request by hinckleyc**
We finished Communication in the conduct.md file. 

Please let me know if we need to fix and errors.

### CODE REVIEW

**Review by ibarrezueta — DISMISSED** [review]

**Comment by ibarrezueta** [comment]
conduct.md:15: MD005 Inconsistent indentation for list items at the same level
458conduct.md:18: MD005 Inconsistent indentation for list items at the same level
459conduct.md:23: MD012 Multiple consecutive blank lines

**Comment by gkapfham** [comment]
Hello @hinckleyc, this branch is out of date with the master base branch. Can you or one of your eam members please update it? We cannot merge this into the master branch until everything is properly updated and the build is still passing correctly. Thanks!

**Comment by gkapfham** [comment]
Hi @hinckleyc, please make sure that you assign four reviewers for this PR! We cannot merge the PRs until all four reviewers sign off on the content. Thanks!

**Review by enpuyou — CHANGES_REQUESTED** [review]
This PR is not up to standard as it does not conform to the conduct style guide

**Comment by gkapfham** [comment]
Hi @hinckleyc, this branch has conflicts! Can you please resolve them today and ensure that this PR is merged if you have plans for it to be merged?

**Comment by koscinskic** [comment]
Corrections and revisions are underway and should be resolved soon

File: conduct.md
Comment (enpuyou) [inline conduct.md:43]: You will also want to add a paragraph describing this sub-section

**Review by enpuyou — CHANGES_REQUESTED** [review]

**Review by tmgcassidy — APPROVED** [review]
I have fixed all my Travis errors, and I have updated all outstanding formatting and content issues.

**Review by koscinskic — APPROVED** [review]

**Review by stephensonc — APPROVED** [review]

**Review by enpuyou — APPROVED** [review]

**Pull request merged**
